### PR TITLE
docs: Initial draft of the Getting Started docs

### PR DESCRIPTION
**Pull request by jimmarino**
## What this PR changes/adds

Adds getting started for adopters. WIP - we will adjust links and other issues later.

## Why it does that

documentation

### CODE REVIEW

File: developer/wip/for-adopters/Getting Started.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
+Many organizations face the challenge of securely sharing data with their partners or other trusted third parties. In
+the past, this has been the realm of proprietary EDI solutions. EDC is an alternative to these systems built on the
+concept of [dataspaces](https://dataspace.eclipse.org/). EDC is a set of components that enable developers to create dataspaces using the following
+building blocks:
+
+- **Identity service** for managing and verifying organizational credentials using `dids`
```
Comment (paullatzelsperger) [inline developer/wip/for-adopters/Getting Started.md:6]:
```suggestion
- **Identity service** for managing and verifying organizational credentials using [DIDs](https://www.w3.org/TR/did-core/)
```

File: developer/wip/for-adopters/Getting Started.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+  and `W3C Verifiable Credentials` or OAuth2 tokens.
```
Comment (paullatzelsperger) [inline developer/wip/for-adopters/Getting Started.md:7]:
```suggestion
  and [W3C Verifiable Credentials](https://www.w3.org/TR/vc-data-model/) or OAuth2 tokens.
```

File: developer/wip/for-adopters/Getting Started.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+- **Catalog service** for publishing and securing assets that can be shared with other organizations.
+- **Negotiation service** for the automated creation and processing of data usage agreements that grant access to data.
+- **Data transfer and monitoring services** for automatically initiating and managing data transfers using off-the-shelf
+  protocols such as `HTTP`, `Kafka`, cloud object storage, or virtually any other technology.
```
Comment (paullatzelsperger) [inline developer/wip/for-adopters/Getting Started.md:11]:
```suggestion
  protocols such as HTTP, Kafka, cloud object storage, or virtually any other technology.
```

File: developer/wip/for-adopters/Getting Started.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+
+## The MVD
+
+The [EDC Minimal Viable Dataspace (MVD)](https://github.com/eclipse-edc/MinimumViableDataspace) sets up and runs a
+complete demonstration dataspace between two organizations. The MVD includes automated setup of a complete dataspace
+environment in a few minutes.
+
+## Overview: Key Components
+
+EDC is architected as modules called ***extensions*** that can be combined and customized to create ***components***
```
Comment (paullatzelsperger) [inline developer/wip/for-adopters/Getting Started.md:45]:
```suggestion
EDC is architected as modules called _extensions_ that can be combined and customized to create _components_
```

File: developer/wip/for-adopters/Getting Started.md [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+that perform specific tasks. These components (the "C" in EDC) are **not** what is commonly referred to as "
+microservices." Rather, EDC components may be deployed as separate services or collocated in a runtime process. This
+section provides a quick overview of the key EDC components.
```
Comment (paullatzelsperger) [inline developer/wip/for-adopters/Getting Started.md:48]: Suggestion: I generally like the analogy of a "toolbox of modules".

File: developer/wip/for-adopters/Getting Started.md [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+
+### The Connector
+
+The Connector is a pair of components that *control* data sharing and *execute* data transfer. These components are the
```
Comment (paullatzelsperger) [inline developer/wip/for-adopters/Getting Started.md:52]:
```suggestion
The Connector is a pair of components that _control_ data sharing and _execute_ data transfer. These components are the
```

File: developer/wip/for-adopters/Getting Started.md [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+components may be deployed in a single monolith (for simple use cases) or provisioned as clusters of individual
+services. It is recommended to separate the Control Plane and Data Plane so they can be individually managed and scaled.
+
+![[Control and Data Planes.svg]]
+
+#### The Control Plane
+
+The Control Plane is responsible for creating contract agreements that grant access to data, managing data transfers,
+and monitoring usage policy compliance. For example, a data consumer's control Plan initiates a contract negotiation
+with a data provider's connector. The negotiation is an asynchronous process that results in a *contract agreement* if
```
Comment (paullatzelsperger) [inline developer/wip/for-adopters/Getting Started.md:63]:
```suggestion
with a data provider's connector. The negotiation is an asynchronous process that results in a _contract agreement_ if
```

File: developer/wip/for-adopters/Getting Started.md [hunk 8]
```diff
@@ -0,0 +1,123 @@
...
+### Federated Catalog
+
+The Federated Catalog (FC) is responsible for crawling and caching data catalogs from other participants. The FC builds
+a local cache that can be queried or processed without resorting to complex distributed queries across multiple
+participants.
+
+### Identity Hub
+
+The Identity Hub securely stores and manages W3C Verifiable Credentials, including the presentation of VCs and the
+issuance and re-issuance process.
```
Comment (paullatzelsperger) [inline developer/wip/for-adopters/Getting Started.md:86]: maybe worth mentioning that it is the reference implementation of DCP

File: developer/wip/for-adopters/Getting Started.md [hunk 9]
```diff
@@ -0,0 +1,123 @@
...
+multiple ReplicaSets or clusters). The connector components can be further decomposed. For example, multiple control
+plane components can be deployed within an organization in a federated manner where departments or subdivisions manage
+specific instances termed `Management Domains`.
+
+### Customizing the EDC
+
+EDC was designed with the philosophy that one size does not fit all. Before deploying an EDC-powered data sharing
+ecosystem, you'll need to build customizations and bundle them into one or more distributions. Specifically:
+
+- **Policies** - Create a set of policies for data access and usage control. EDC adopts a code-first approach, which
```
Comment (paullatzelsperger) [inline developer/wip/for-adopters/Getting Started.md:110]: once the rest of the Adopter document is ready, we should add links here

File: developer/wip/for-adopters/Getting Started.md [hunk 10]
```diff
@@ -0,0 +1,123 @@
...
+  involves writing *policy functions*.
+- **Verifiable Credentials** - Define a set of W3C Verifiable Credentials for your use cases that your policy functions
+  can process. For example, a credential that identifies a particular partner type.
+- **Data transfer types** - Define a set of data transfer technologies or types that must be supported. For example,
+  choose out-of-the-box support for HTTP, S3-based transfers, or Kafka. Alternatively, you can select your preferred
+  wire protocol and implement a custom data plane.
+- **Backend connectivity** - You may need to integrate EDC components with back-office systems. This is done by writing
+  custom extensions.
+
+Third parties and other open source projects distribute EDC extensions that can be included in a distribution. These
```
Comment (paullatzelsperger) [inline developer/wip/for-adopters/Getting Started.md:120]: I'm assuming at some point we'll have a definition of terms, and when we do, we should add a link from "distribution" to it
```suggestion
Third parties and other open source projects distribute EDC extensions that can be included in a [distribution](). These
```

**Review by paullatzelsperger — APPROVED** [review]
channeling my inner Monk...

File: developer/wip/for-adopters/Getting Started.md (hunk 8 above)
Comment (jimmarino) [reply developer/wip/for-adopters/Getting Started.md:86]: Let's do this in a top section later with DSP as well

File: developer/wip/for-adopters/Getting Started.md (hunk 10 above)
Comment (jimmarino) [reply developer/wip/for-adopters/Getting Started.md:120]: Yes, we should have a glossary once we get done.

**Pull request merged**